lib/server.js

```diff
@@ -122,7 +124,21 @@ function createServer (options, httpHandler) {
     this.ready(listenCallback.call(this, server, listenOptions))
   }
 
-  return { server, listen }
+  const serverHasCloseAllConnections = typeof server.closeAllConnections === 'function'
+  const serverHasCloseIdleConnections = typeof server.closeIdleConnections === 'function'
+  const serverHasCloseHttp2Sessions = typeof server.closeHttp2Sessions === 'function'
+
+  let forceCloseConnections = options.forceCloseConnections
+  if (forceCloseConnections === 'idle' && !serverHasCloseIdleConnections) {
+    throw new FST_ERR_FORCE_CLOSE_CONNECTIONS_IDLE_NOT_AVAILABLE()
+  } else if (typeof forceCloseConnections !== 'boolean') {
+    /* istanbul ignore next: only one branch can be valid in a given Node.js version */
```

Comment:
istanbul? we can remove that, as we dont use istanbul or nyc. We use c8. Doesnt matter